src/app/components/client/DataBrokerProfiles.tsx

```diff
@@ -10,12 +10,20 @@ import { useL10n } from "../../hooks/l10n";
 import IconChevronDown from "./assets/icon-chevron-down.svg";
 import { useState } from "react";
 import { OnerepScanResultRow } from "knex/types/tables";
-import { OpenInNew } from "../server/Icons";
+import { DataBrokerNames } from "../../functions/universal/dataBrokerNames";
 
 export type Props = {
   data: OnerepScanResultRow[];
 };
 
+function getDataBrokerName(dataBrokerName: string) {
+  const result = DataBrokerNames.find(
+    ({ data_broker }) => data_broker === dataBrokerName,
+  );
+
+  return result?.data_broker_pretty || dataBrokerName;
```

Comment:
Shouldn't cause a problem in practice, but a good habit to get into is to use `??` if you're not comparing booleans, to avoid issues if you're ever doing this for something that could be e.g. an empty string, or 0:

```suggestion
  return result?.data_broker_pretty ?? dataBrokerName;
```

Reply:
😮‍💨  Revised in 0c13bf7944473b62dd05634ffbe0af9804cb8410